clash-lib/src/Clash/Backend.hs

```diff
@@ -149,7 +149,7 @@ class HasIdentifierSet state => Backend state where
   -- | getSrcSpan
   getSrcSpan       :: State state SrcSpan
   -- | Block of declarations
-  blockDecl        :: Identifier -> [Declaration] -> Ap (State state) Doc
+  blockDecl        :: Identifier -> [CompDecl] -> [Declaration] -> Ap (State state) Doc
```

Comment:
We should probably document the arguments

Reply:
One aspect of the documentation should be that `CompDecl` is only used in VHDL.

Reply:
Yes!